src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningData.cshtml

```diff
@@ -0,0 +1,128 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.PriorLearningDataViewModel
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@{
+    ViewData["Title"] = "Add recognition of prior learning details";
+    ViewBag.ValidationOrder = "TrainingTotalHours,DurationReducedByHours,IsDurationReducedByRpl,ReducedDuration,CostBeforeRpl,ReducedPrice";
```

Comment:
I believe that the standard error partial orders errors by the order of the fields in the viewmodel, though this might have changed since I worked on it. Worth checking though.